fastapi_rest_framework/sqlalchemy/interactor.py

```diff
@@ -0,0 +1,30 @@
+import typing
+
+import saritasa_sqlalchemy_tools
+
+from .. import interactors, permissions
+from . import repositories
+
+
+class SqlAlchemyInteractor(
+    interactors.ApiDataInteractor[
+        permissions.UserT,
+        saritasa_sqlalchemy_tools.SelectStatement[
+            saritasa_sqlalchemy_tools.BaseModelT
+        ],
+        repositories.SqlAlchemyRepositoryT,
+        saritasa_sqlalchemy_tools.BaseModelT,
+    ],
+    typing.Generic[
+        permissions.UserT,
+        saritasa_sqlalchemy_tools.BaseModelT,
+        repositories.SqlAlchemyRepositoryT,
+    ],
+):
+    """Interactor for sqlalchemy."""
+
+
+class SqlAlchemyInteractorHooksMixin(
+    interactors.BaseHooksMixin[saritasa_sqlalchemy_tools.BaseModelT],
+):
+    """Hooks for sqlalchemy."""
```

Comment:
This class is really needed? Why not just use `BaseHooksMixin`?

Reply:
Just to specify that it work with models from sqlachemy-tools(for now it's maybe no difference, but it future it will make it easier to spot changes for devs).